components/Forms/RecipeForm.js

```diff
@@ -90,17 +119,30 @@ export default function RecipeForm({ onSubmit, onDelete, data }) {
             router.back();
           }}
         ></IconButton>
-        {imageUrl && (
+        {preview && (
           <StyledImageCloudinary
-            src={imageUrl}
+            src={
+              preview.imageUrl || "/img/jason-briscoe-7MAjXGUmaPw-unsplash.jpg"
+            }
             alt="Uploaded Image"
             width={100}
             height={300}
           />
         )}
-        <StyledImageUploadContainer>
-          <Plus width={40} height={40} />
-          <StyledImageUpload type="file" onChange={uploadImage} />
+
+        <StyledImageUploadContainer htmlFor="upload">
+          <form onSubmit={uploadImage}>
+            {/* <IconButtonLarge style="plus" /> */}
+            {/* <Plus width={40} height={40} /> */}
+            <input
+              type="file"
+              name="file"
+              id="upload"
+              onChange={handleImage}
+            ></input>
```

Comment:
can be self-closing:

```suggestion
            />
```